bin/cron/calculate_workshop_survey_results

```diff
@@ -18,21 +18,18 @@ def main
   end
 
   course_scores['CSP Local Summer Workshops'] = summarize_workshop_surveys(
-    Pd::Workshop.in_state(Pd::Workshop::STATE_ENDED).where(course: Pd::Workshop::COURSE_CSP, subject: Pd::Workshop::SUBJECT_CSP_SUMMER_WORKSHOP),
-    nil,
-    false
+    workshops: Pd::Workshop.in_state(Pd::Workshop::STATE_ENDED).where(course: Pd::Workshop::COURSE_CSP, subject: Pd::Workshop::SUBJECT_CSP_SUMMER_WORKSHOP),
```

Comment:
Any reason in particular why this is the only one with the `workshops` named param?